C++/Navio/Navio2/RCOutput_Navio2.cpp

```diff
@@ -0,0 +1,26 @@
+#include "RCOutput_Navio2.h"
+
+RCOutput_Navio2::RCOutput_Navio2()
+{
+
+}
+
+bool RCOutput_Navio2::initialize()
+{
+    if (!pwm.init(PWN_OUTPUT)) {
+        fprintf(stderr, "Output Enable not set. Are you root?\n");
```

Comment:
Doesn't make sense. We don't have Output enable on Navio2.

Reply:
Sorry, but I can't agree with you. Navio2 have Output enable